docs/reference/remap/encode_json.cue

```diff
@@ -0,0 +1,32 @@
+package metadata
+
+remap: functions: encode_json: {
+	_all_types: ["boolean", "integer", "float", "string", "timestamp", "regex", "null"]
+
+	arguments: [
+		{
+			name:        "value"
+			description: "The value to return a json representation of."
+			required:    true
+			type:        _all_types
+		},
+	]
+	return: ["string"]
+	category: "coerce"
```

Comment:
Maybe we need a new category like `encode`, `serialize`, or `marshal`?

Reply:
+1 on this. I would vote for `encode` on this, which feels a teensy bit less like a lower-level programming operation than `serialize` or `marshal`. Just a gut feeling, though.